run.py

```diff
@@ -1,10 +1,16 @@
 import genesis.streaming as gs
 
-message = { 'hello': 'world' }
+message = {"hello": "world"}
 
-with gs.open("kafka://localhost/test1", "rw", start_at="earliest", format="json", config="kafkacat.conf") as stream:
-	stream.write(message)
+with gs.open(
+    "kafka://localhost/test1",
+    "rw",
+    start_at="earliest",
+    format="json",
+    config="kafkacat.conf",
+) as stream:
+    stream.write(message)
 
-#with gs.open("kafka://localhost/test1", "r", start_at="earliest", format="json") as stream:
-	for _, rec in stream(timeout=10):
-		print(rec)
+    # with gs.open("kafka://localhost/test1", "r", start_at="earliest", format="json") as stream:
+    for _, rec in stream(timeout=10):
+        print(rec)
```

Comment:
This whole file is a quick hack for testing, which should go away once a test suite is in place.